File: src/shared/classes/ExplicitContent.ts
export class ExplicitContent {
  filter_enabled: boolean
  filter_locked: boolean

  static from(json){
    if (json === null) return null

    let explicitContent = new ExplicitContent()

    explicitContent.filter_enabled = json?.filter_enabled
    explicitContent.filter_locked = json?.filter_locked

    return explicitContent
  }
}

File: src/main/java/de/wittenbude/exportify/models/embeds/ExplicitContent.java
package de.wittenbude.exportify.models.embeds;

import lombok.Getter;
import lombok.Setter;
import lombok.experimental.Accessors;

@Getter
@Setter
@Accessors(chain = true)
public class ExplicitContent {
    private Boolean filterEnabled;
    private Boolean filterLocked;
}
